ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -351,6 +351,11 @@ export default defineComponent({
       }
     )
 
+    // When a change is made to homeOwners, check if any actions have changed, if so set flag
+    watch(() => props.homeOwners, () => {
+      setUnsavedChanges(props.homeOwners.some(owner => !!owner.action))
+    })
+
```

Comment:
Keep in mind this will toggle the flag regardless of filing implementation! 
So if that is intended for both Transfers and Registrations than 👍